doc/26_Best_Practice/60_Modifying_Permissions_based_on_Object_Data.md

```diff
@@ -36,7 +36,7 @@ services:
         arguments:
             - '@Pimcore\Security\User\UserLoader'
         tags:
-            - { name: kernel.event_listener, event: pimcore.admin.object.get.preSendData, method: checkPermissions }
+            - { name: kernel.event_listener, event: pimcore.admin.dataObject.get.preSendData, method: checkPermissions }
```

Comment:
```suggestion
            - { name: kernel.event_listener, event: pimcore.admin.dataobject.get.preSendData, method: checkPermissions }
```
not sure if is case sensitive, but noticed that is lowercased

https://github.com/pimcore/admin-ui-classic-bundle/commit/4942d2d2be4c5ac210b8c91cf87e83446a8893db

Reply:
Can you please also rebase the PR to be on `11.0` base branch?